### DISCUSSION TITLE
Will this still work with. .net core 3.1?

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#11416] [Q&A] [answered]
I last used OC when it was in RC2 and was wondering if the latest 1.0 will work with .net core 3.1?

**Comment by Skrypt** [comment, marked as answer]
1.3 doesn't support 3.1 anymore.

**Comment by hishamco** [comment]
If you are using <=1.2.2 you can able to use .NET Core 3.1
